src/components/navigation/document-tab-panel.tsx

```diff
@@ -176,6 +176,8 @@ export class DocumentTabPanel extends BaseComponent<IProps, IState> {
           currentClassHash={classHash}
           currentTeacherName={user.name}
           currentTeacherId={user.id}
+          subTab={subTab}
+          problem={this.stores.problem.title}
```

Comment:
In the code we often use `problem` as shorthand for the problem ordinal, e.g. "1.2". If this is meant to refer to the title moving forward you might want to name it `problemTitle` or `problemName` to avoid ambiguity. If this is strictly temporary on the way to displaying thumbnails, then obviously you shouldn't worry about it.